### ISSUE TITLE
SPIKE - Trigger jobs based on data availability

### ISSUE THREAD

**Issue by tech3371** [IMAP-Science-Operations-Center/sds-data-manager#97]
## Description

Short description of the issue you would like to solve. 

## Requirements
- Requirement 1
- Requirement 2

### Nice to have or Goal Requirements
- Nice to have 1
- Nice to have 2

## Additional notes
Any additional notes on the issue, if you have them. 

## Related tickets
- `#123` 

## Follow up tickets
This spike is not considered complete until at least one follow up issue is created. 
- [ ] Placeholder ticket

**Below is the template for the response to this ticket. Add as many solutions as needed, but preferably include 2-5 for discussion. The response should be posted as a comment on this issue, or linked in a comment. **

# Solution 1
Write an overview of the solution here.

## Pros:
- pro 1

## Cons:
- con 1

## Additional notes:
- note

# Solution 2
Write an overview of the solution here.

## Pros:
- pro 1

## Cons:
- con 1

## Additional notes:
- note

# Summary
Write up a summary of your findings, including your preferred solution.

**Comment by maxinelasp** [comment]
Blocked on #71

**Comment by maxinelasp** [comment]
Currently in PR with #191, with related ticket #187. Completed by @laspsandoval https://lasp.colorado.edu/galaxy/display/IMAP/SDC+Processing+Architecture+Overview

**Comment by sdhoyt** [comment]
SPIKE is closed by [the comment above](https://github.com/IMAP-Science-Operations-Center/sds-data-manager/issues/97#issuecomment-1791014940). SPIKE was re-opened for a data re-processing discussion, but that discussion has been pushed to SPIKE #187
